src/dso-version-selector.js

```diff
@@ -0,0 +1,73 @@
+(function() {
+  'use strict';
+  // Get list of versions from versions.json
+  var oReq = new XMLHttpRequest();
+  oReq.responseType = "json";
+  oReq.addEventListener("load", createSelector);
+  oReq.open("GET", "/versions.json");
+  oReq.send();
+
+  function createSelector () {
+    // Create input select template
+    var jsonVersions = this.response;
+    var selectTemplate = document.createElement("select");
+    var selectId = document.createAttribute("id");
+    var selectClass = document.createAttribute("class");
+    var selectOnChange = document.createAttribute("onchange");
+    selectId.value = "dsoVersionSelector";
+    selectClass.value = "dso-version-selector";
+    selectOnChange.value = "openVersion()";
+    selectTemplate.setAttributeNode(selectId);
+    selectTemplate.setAttributeNode(selectClass);
+    selectTemplate.setAttributeNode(selectOnChange);
```

Comment:
Bovenstaande ~11 regels code kunnen netter:
* Opdelen in blokken
* De code voor het opbouwen van een element bij de code van het maken van het element voegen
* Double quotes, de voorkeur heeft single quotes

Op regel 4 wordt een variable `oReq` gemaakt. de `o` staat voor `object`, je kan aan de naam van de variable zien wat het type is. Dit is een naamgevingsconventie die wij niet gebruiken. Wij gebruiken camelCased, volledig uitgeschreven woorden. `versionsRequest` klinkt wel aardig?